src/overlays/actors/ovl_En_Tanron4/z_en_tanron4.h

```diff
@@ -8,11 +8,25 @@ struct EnTanron4;
 typedef void (*EnTanron4ActionFunc)(struct EnTanron4*, GlobalContext*);
 
 typedef struct EnTanron4 {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0xE8];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ s16 randRollTimer;
+    /* 0x146 */ s16 timers[2];
+    /* 0x14A */ u8 flyState;
+    /* 0x14C */ Vec3f targetPos;
+    /* 0x158 */ s16 step;
+    /* 0x15A */ s16 roll;
+    /* 0x15C */ s16 rollTarget;
+    /* 0x160 */ f32 timeInfluence;
+    /* 0x164 */ SkelAnime skelAnime;
+    /* 0x1A8 */ Vec3s jointTable[11];
+    /* 0x1EA */ Vec3s morphTable[11];
```

Comment:
Since you are already using the limb enum, you should use it here too
```suggestion
    /* 0x1A8 */ Vec3s jointTable[OBJECT_TANRON4_LIMB_MAX];
    /* 0x1EA */ Vec3s morphTable[OBJECT_TANRON4_LIMB_MAX];
```